docs/MIGRATING-v0.11.0.md

```diff
@@ -0,0 +1,452 @@
+
+<table width="100%">
+<thead><tr><td >Before</td><td>After</td></tr></head>
+<tr valign="top"><td>
+
+<!-- BEFORE -->
+```go
+assert.EventRecorded(SomeEvent{ /* ... */ })
+```
+
+</td><td>
+
+<!-- AFTER -->
+```go
+ToRecordEvent(SomeEvent{ /* ... */ })
+```
+
+</td></tr>
+<tr><td></td><td></td></tr>
+<tr valign="top"><td>
+
+<!-- BEFORE -->
+```go
+assert.EventTypeRecorded(SomeEvent{})
+```
+
+</td><td>
+
+<!-- AFTER -->
+```go
+ToRecordEventOfType(SomeEvent{})
+```
+
+</td></tr>
+<tr><td></td><td></td></tr>
+<tr valign="top"><td>
+
+<!-- BEFORE -->
+```go
+assert.CommandExecuted(SomeCommand{ /* ... */ })
+```
+
+</td><td>
+
+<!-- AFTER -->
+```go
+ToExecuteCommand(SomeCommand{ /* ... */ })
+```
+
+</td></tr>
+<tr><td></td><td></td></tr>
+<tr valign="top"><td>
+
+<!-- BEFORE -->
+```go
+assert.CommandTypeExecuted(SomeCommand{})
+```
+
+</td><td>
+
+<!-- AFTER -->
+```go
+ToExecuteCommandOfType(SomeCommand{})
+```
+
+</td></tr>
+<tr><td></td><td></td></tr>
+<tr valign="top"><td>
+
+<!-- BEFORE -->
+```go
+assert.AllOf(/* ... */)
+```
+
+</td><td>
+
+<!-- AFTER -->
+```go
+AllOf(/* ... */)
+```
+
+</td></tr>
+<tr><td></td><td></td></tr>
+<tr valign="top"><td>
+
+<!-- BEFORE -->
+```go
+assert.AnyOf(/* ... */)
+```
+
+</td><td>
+
+<!-- AFTER -->
+```go
+AnyOf(/* ... */)
+```
+
+</td></tr>
+<tr><td></td><td></td></tr>
+<tr valign="top"><td>
+
+<!-- BEFORE -->
+```go
+assert.NoneOf(/* ... */)
+```
+
+</td><td>
+
+<!-- AFTER -->
+```go
+NoneOf(/* ... */)
+```
+
+</td></tr>
+<tr><td></td><td></td></tr>
+<tr valign="top"><td>
+
+<!-- BEFORE -->
+```go
+assert.Should(
+    "do something",
+    func(s *assert.S) { /* ... */},
+)
+```
+
+</td><td>
+
+<!-- AFTER -->
+```go
+ToSatisfy(
+    "do something",
+    func(t *SatisfyT) { /* ... */ },
+)
+```
+
+</td></tr>
+</table>
+
+### Enabling and disabling message handlers
+
+Within a `Test` both projection and integration message handler types are
+disabled by default. Prior to this release such handlers could be enabled using
+`WithOperationOptions(...)`.
+
+As of this release enabling or disabling handlers by type is discouraged
+<sup>2</sup>. Instead, individual handlers are enabled and disabled by name
+using the `Test.EnableHandlers()` and `DisableHandlers()` methods.
+
+This change is made to accomodate future changes to the expectation and
+reporting systems that will analyse each handler's routing configuration to
+eliminate impossible expectations and provide more meaningful failure reports.
+
+<table width="100%">
+<thead><tr><td>Before</td><td>After</td></tr></head>
+<tr valign="top"><td>
+
+<!-- BEFORE -->
+```go
+test := New().Begin(
+    t,
+    WithOperationOptions(
+        engine.EnableProjections(true),
+    ),
+)
+```
+
+</td><td>
+
+<!-- AFTER -->
+```go
+test := Begin(t).
+    EnableHandlers("some-projection")
+```
+
+</td></tr>
+</table>
+
+---
+
+<small><strong><sup>1</sup></strong> The function passed to `Call()` no longer
```

Comment:
These `<small>` things don't seem to work. But that's probably ok.

Reply:
Hrm, yeah.  They work when rendering a preview in VS code, but not on GitHub :/

I might just remove them if they don't do anything; otherwise, they are just noise.